casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala

```diff
@@ -257,7 +250,14 @@ object InterpreterUtil {
                        invalidBlocks,
                        isGenesis //should always be false
                      )
-    } yield replayResult
+    } yield replayResult.leftMap[Throwable] {
```

Comment:
I think `raiseError` would be preferable to `rethrow` here.

Reply:
I don't see how, and am having a hard time writing sth similarly concise. I guess it'd be most efficient if you changed that later if you have a better solution.